ddtrace/contrib/internal/ray/patch.py

```diff
@@ -287,6 +287,28 @@ def traced_actor_method_call(wrapped, instance, args, kwargs):
         return wrapped(*args, **kwargs)
 
 
+def traced_get(wrapped, instance, args, kwargs):
+    """
+    Trace the calls of ray.get
+    """
+    if tracer.current_span() is None:
```

Comment:
I added a guard for the config in traced_put and traced_wait. Can you do the same here please ?